app/console

```diff
@@ -3,28 +3,36 @@
 
 use Symfony\Bundle\FrameworkBundle\Console\Application;
 use Symfony\Component\Console\Input\ArgvInput;
-use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Debug\Debug;
-
-// if you don't want to setup permissions the proper way, just uncomment the following PHP line
-// read http://symfony.com/doc/current/setup.html#checking-symfony-application-configuration-and-setup
-// for more information
-//umask(0000);
+use Symfony\Component\Dotenv\Dotenv;
```

Comment:
it will break prod, as dotenv is in require-dev section

Reply:
it won't because the Dotenv class won't be constructed in prod at all if the `APP_ENV` var is set, and it must be set to `prod` in prod env